docs/root/version_history/current.rst

```diff
@@ -40,6 +40,7 @@ Changes
   tracing is not forced.
 * router: allow retries of streaming or incomplete requests. This removes stat `rq_retry_skipped_request_not_complete`.
 * router: allow retries by default when upstream responds with :ref:`x-envoy-overloaded <config_http_filters_router_x-envoy-overloaded_set>`.
+* runtime: add new  gauge `deprecated_feature_seen_since_process_start` that gets reset across hot restarts.
```

Comment:
nit: can you ref link to the section where this is explained?

Reply:
Done